.github/workflows/integration-tests.yaml

```diff
@@ -57,24 +77,12 @@ jobs:
           }
           EOT
       - name: Install gotestsum
-        run: go install gotest.tools/gotestsum@latest
-        shell: bash
-      # The GitHub token we generated is valid for an hour. The full test suite can run for longer than an hour.
-      # So we first run the git tests while the token is fresh, and the run the rest of the tests.
-      - name: Test Git integration
-        run: go test -v -timeout 0 -run "^\\QTestProjectWithGitUsernameExport\\E$" integration_test.go
+        run: |
+          sudo apt-get update
```

Comment:
The original workflow used `go install gotest.tools/gotestsum@latest`, but for reasons I didn't debug, it stopped working with the multiple nodes. It is probably something to do with the path. But I'll swap this for a dedicated action to install `gotestsum` rather than use OS specific install processes.